### DISCUSSION TITLE
use Copilot as student

### DISCUSSION THREAD

**Discussion by lcouedor** [community/community#35432] [Copilot Conversations]
Hello,

I wanted to try using copilot on vscode, so i've installed the extension and now need to register. I've added my school email to my account, and now have to upload a document to prove my status. I've uploaded a school certificate where you can see my name, the name of the school, and adequate date, but I can't go any further because of this error : 
<img width="738" alt="image" src="https://user-images.githubusercontent.com/83167701/194555295-779bb384-c602-4dee-87b3-f26c6f86c121.png">

It also sometimes ask me to allow my browser to use my location data, but even enabled it doesn't change anything.

Does anyone have a solution to this problem ?

**Comment by iamcjwilliams** [comment]
@lcouedor please visit here for more information regarding your issue [GitHub Global Campus](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student)
